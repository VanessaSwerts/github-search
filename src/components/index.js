import CustomHeader from './Header/Header'
import CustomFooter from './Footer/Footer'
import Error from './Error/Error'
import Loading from './Loading/Loading'

export {
  CustomHeader,
  CustomFooter,
  Error,
  Loading
}