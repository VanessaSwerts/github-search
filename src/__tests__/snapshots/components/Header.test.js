import React from 'react'
import { render } from '@testing-library/react'
import { CustomHeader } from '../../../components'

describe('Testes de Snapshot da Header', () => {

  test('Renderização da tela Header', () => {
    const { asFragment } = render(<CustomHeader />)

    expect(asFragment(<CustomHeader />)).toMatchSnapshot()
  })

}) 