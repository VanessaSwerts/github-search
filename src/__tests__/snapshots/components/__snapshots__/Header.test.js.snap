// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testes de Snapshot da Header Renderização da tela Header 1`] = `
<DocumentFragment>
  <div
    class="navbar"
  >
    <i
      class="fab fa-github"
    />
    <span>
      GitHub Search
    </span>
  </div>
</DocumentFragment>
`;
